features/progress.yml

```
@@ -1,8 +1,9 @@
-draft_date: 2024-06-05
 name: <progress>
-description: The `<progress>` element TODO.
+description: The `<progress>` element displays an indicator showing the completion progress of a task, typically displayed as a progress bar.
 spec: https://html.spec.whatwg.org/multipage/form-elements.html#the-progress-element
-group: forms
+group:
+  - forms
+  - html-elements
 caniuse: progress
```

I'm pretty skeptical of the Safari iOS mismatch for the DOM interface and the element but it's very historical and AFAICT impossible to test now. Let's do this to align to caniuse:

```suggestion
caniuse: progress
compute_from: html.elements.progress
```